.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [7.1, 7.2, 7.3, 7.4]
+        stability: [prefer-lowest, prefer-stable]
+
+    name: PHP ${{ matrix.php }} - ${{ matrix.stability }} Test
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: curl, sodium
```

Comment:
Don't add `curl` here. it is already enabled by default.

Reply:
Why not adding it as a dependency as it‘s a requirement of the library? Its not breaking anything and ensures it‘s installed if something changes in the php installer.

This is also added in other big packages like laravel‘s, it‘s not required their too, but still explicitly added in the workflows